ucc-demo.ipynb

```diff
@@ -0,0 +1,319 @@
+    "\n",
+    "Unitary Compiler Collection (`ucc`) is a Python library for frontend-agnostic, high performance compilation of quantum circuits. UCC's goal is to gather together the best of open source compilation to make quantum programming simpler, faster, and more scalable. For more details, click the links below and read the `ucc` launch [blogpost](https://unitary.foundation/posts/2025_ucc_launch_blog/)\n",
+    "\n",
+    "[![Repository](https://img.shields.io/badge/GitHub-5C5C5C.svg?logo=github)](https://github.com/unitaryfund/ucc)\n",
+    "[![Unitary Foundation](https://img.shields.io/badge/Supported%20By-Unitary%20Foundation-FFFF00.svg)](https://unitary.foundation)\n",
+    "[![Documentation Status](https://readthedocs.org/projects/ucc/badge/?version=latest)](https://ucc.readthedocs.io/en/latest/?badge=latest)\n",
+    "[![Discord Chat](https://img.shields.io/badge/dynamic/json?color=blue&label=Discord&query=approximate_presence_count&suffix=%20online.&url=https%3A%2F%2Fdiscord.com%2Fapi%2Finvites%2FJqVGmpkP96%3Fwith_counts%3Dtrue)](http://discord.unitary.foundation)\n",
+    "\n",
+    "The goal of this tutorial is to give you an overview of `ucc` features, and provide an easy entry to test `ucc` on your existing quantum programs, and to explore developing custom compiler passes.\n",
+    "\n",
+    "Note that `ucc` is in its early stages, building mostly on the shoudlers of other libraries like qiskit and qbraid. This notebook gives you a flavor of where `ucc` is heading, but there's plenty of work ahead to drive utility and adoption. Early feedback is precious to ensuring that direction will be fruitful."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Setup\n",
+    "\n",
+    "### GitHub [Codespaces](https://docs.github.com/en/codespaces/developing-in-a-codespace/creating-a-codespace-for-a-repository#creating-a-codespace-for-a-repository)\n",
+    "\n",
+    "   *To use Codespaces, an active Github account is required.*\n",
+    "\n",
+    "1. In the repository, select the green Code button, switch to the Codespaces tab, and click the `Create codespace on main` button.\n",
+    "\n",
+    "![image](https://github.com/user-attachments/assets/23d1004b-fc28-4717-a9e2-39e579fa2d35)\n",
+    "\n",
+    "2. Codespaces may take a few moments to load while creating the codespace. Once complete, you will be able to select the `demo/ucc-demo.ipynb` notebook from the explore list on the left hand side.\n",
+    "\n",
+    "3. When you first execute the notebook, you will first be prompted to select  `Python Environments...` and then the recommended environment from the `Select Another Kernel` window.\n",
+    "\n",
+    "### Local \n",
+    "If you are running this notebook locally, be sure you have have the `ucc` packaged installed via `pip install ucc` or have followed the [documentation](https://ucc.readthedocs.io/en/latest/#installation) on setting up via poetry."
+   ]
+  },
```

Comment:
When you are already in codespace this section feels superfluous but then you do need to run the code in the Local section in order to import ucc. Perhaps move the codespaces part to the readme and then start off everyone right away with the import code in the Local code block?

Reply:
Yes that makes sense, I'll move these to the README.